.github/workflows/test.yml

```
@@ -52,7 +52,8 @@ jobs:
 
       - name: <Deploy> Submit coverage results to Coveralls
         if: runner.os == 'Linux'
-        run: coveralls --service=github
+        # Coveralls にアップロード失敗してもCIは継続させる
+        run: coveralls --service=github || echo "::warning::Coveralls upload failed, but continuing"
```

他の理由で失敗してもこのメッセージになるのがちょっと不安かもですね･･･！
意図は「アップロードへの失敗を無視する」、実装は「全部無視する」なので、こうしますか！

```suggestion
        # Coveralls へのアップロードが失敗しても CI は継続させる
        run: coveralls --service=github || echo "::warning::Coveralls failed, but continuing"
```